src/geom/rectbin.jl

```diff
@@ -94,14 +94,14 @@ function render(geom::RectangularBinGeometry, theme::Gadfly.Theme, aes::Gadfly.A
 
     allvisible = true
     for c in cs
-        if c == nothing
+        if isnothing(c)
             allvisible = false
             break
         end
     end
 
     if !allvisible
-        visibility = cs .!= nothing
+        visibility = !isnothing.(cs)
```

Comment:
shouldn't the `!` need to be broadcast here?  so `visibility  = .!isnothing.(cs)`.

codvec says this line [isn't covered](https://codecov.io/gh/GiovineItalia/Gadfly.jl/pull/1278/src/src/geom/rectbin.jl).  would be great if you could add a test.

Reply:
Good catch. Fixed. I'm not grocking that code well enough to understand how to trigger it in a test case unfortunately.